### PR TITLE
Add mesalib

### PR DESCRIPTION
**Pull request by ocefpaf**
@doutriaux1 let's see what happens with the `skip` statement here.

### CODE REVIEW

File: recipes/mesalib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.freedesktop.org/pub/mesa/older-versions/7.x/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2b5595952e94d5ecdb7c0a53ac6db9bbf23cc2228b09124ae0c37f9507690ac
+
+build:
+  skip: True  # [win or py3k]
+  number: 0
+  track_features:
+    - mesalib
```
Comment (ocefpaf) [inline recipes/mesalib/meta.yaml]: I am not sure exactly what you are expecting here @doutriaux1. @msarahan what do we need to do to track custom features?

**Comment by ocefpaf** [comment]
@doutriaux1 I don't see the problem you mentioned in https://github.com/conda-forge/staged-recipes/pull/2377

File: recipes/mesalib/meta.yaml (hunk 1 above)
Comment (doutriaux1) [inline recipes/mesalib/meta.yaml]: I have a few packages in the pipeline down the road that will rely on this feature being present or not.

Comment (doutriaux1) [inline recipes/mesalib/meta.yaml]: so basically I need this package to say "I provide the `mesalib` feature.

**Comment by ocefpaf** [comment]
@doutriaux1 let's merge this one because I also fixed some small format issue with the recipe.

**Comment by doutriaux1** [comment]
Thanks!

**Comment by ocefpaf** [comment]
No problem. Now we need to wait for the feedstock.

**Pull request merged**